source/D2CommonDefinitions/include/D2Structs.h

```diff
@@ -5,6 +5,8 @@
 
 #include "D2QuestDataEx.h"
 
+typedef uint32_t D2UnitGUID;
+static const D2UnitGUID D2UnitInvalidGUID = (D2UnitGUID)-1;
```

Comment:
I'd also use `constexpr` there. Adding to that, one could think about using the more modern `using D2UnitGUID = uint32_t;` declaration instead of `typedef`

Reply:
> This static modifier, in this case, would give this variable file-scope storage modifier (unless that is intentional). If this is intended to act as a true constant, you might want to use constexpr.

It was indeed intended, I've switched to constexpr